src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpCodeActions.cs

```diff
@@ -246,18 +231,8 @@ class C
              * outcome for the modified .editorconfig style.
              */
 
-            VisualStudio.SolutionExplorer.BeginWatchForCodingConventionsChange(new ProjectUtils.Project(ProjectName), "Class1.cs");
-            try
-            {
-                VisualStudio.SolutionExplorer.SetFileContents(new ProjectUtils.Project(ProjectName), ".editorconfig", editorConfig.Replace("true:warning", "false:warning"));
-            }
-            finally
-            {
-                VisualStudio.SolutionExplorer.EndWaitForCodingConventionsChange(Helper.HangMitigatingTimeout);
-            }
+            VisualStudio.SolutionExplorer.SetFileContents(new ProjectUtils.Project(ProjectName), ".editorconfig", editorConfig.Replace("true:warning", "false:warning"));
 
-            // Wait for CodingConventions library events to propagate to the workspace
```

Comment:
It's unclear to me if we still need some other wait after SetFileContents to ensure any file change events have been sent through to the workspace and been registered. This existing code wouldn't have been safe for that _either_. I thought I had an old PR on this -- so I still have to dig it out.